### PR TITLE
Add ANARCI

### PR DESCRIPTION
**Pull request by sdvillal**
This is an attempt at packaging ANARCI (Antibody Numbering and Receptor ClassIfication).

Note that at the moment they do not provide release versions, so I have gone with the date of the commit.

### CODE REVIEW

**Comment by sdvillal** [comment]
@BiocondaBot please fetch artifacts

**Comment by sdvillal** [comment]
@BiocondaBot please fetch artifacts

**Comment by sdvillal** [comment]
@BiocondaBot please add label

File: recipes/anarci/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+source:
+  url: https://github.com/oxpig/ANARCI/archive/{{ commit }}.zip
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script:
+    - "{{ PYTHON }} setup.py install"
```
Comment (dpryan79) [inline recipes/anarci/meta.yaml:18]: Can you change this to `{{ PYTHON }} -m pip install . -vv --no-deps` and then remove `hmmer` from the host section?

Comment (sdvillal) [reply recipes/anarci/meta.yaml:18]: I actually tried first with the "pip install" command, but that makes the building process, that generates the HMMs, not work well. I think it probably is an issue upstream.

I will anyway try to see if I can honor some of your suggestions.

Comment (dpryan79) [reply recipes/anarci/meta.yaml:18]: Ah, good to know. If that fails then please just revert. I'll approve the PR so it can get merged either way.

**Comment by sdvillal** [comment]
@dpryan79 I have commented why we need the non-standard install command and moved hmmer from host to build dependency, that I guess conveys better the fact that hmmer is needed both at build and runtime.

Green tick! :-)

Thanks for reviewing.

**Comment by dpryan79** [comment]
@bioconda-bot please fetch artifacts

**Comment by dpryan79** [comment]
@bioconda-bot please merge

Thanks @sdvillal !

**Pull request merged**